modules/passkey/contracts/SafeWebAuthnSignerFactory.sol

```diff
@@ -42,9 +54,25 @@ contract SafeWebAuthnSignerFactory is ISafeSignerFactory {
         uint256 x,
         uint256 y,
         P256.Verifiers verifiers
-    ) external view override returns (bytes4 magicValue) {
-        if (WebAuthn.verifySignature(message, signature, WebAuthn.USER_VERIFICATION, x, y, verifiers)) {
-            magicValue = ERC1271.MAGIC_VALUE;
+    ) external view override returns (bytes4 /*magicValue*/) {
+        address _singleton = SINGLETON;
```

Comment:
```suggestion
        address singleton = SINGLETON;
```

Reply:
The reason I suggested this was that this is the only occurrence of a function local variable starting with `_` in the passkey code. The conversation was marked as resolved without additional context, so I just want to understand why you think we should keep it this way.

Reply:
I added the change to code here: https://github.com/safe-global/safe-modules/pull/370/files#diff-ce13df21c642f8e2ad855020959aff44c4035a27feb2c9e478cb7f96be9d620dR58